addons/point_of_sale/static/src/xml/Screens/ProductScreen/ProductItem.xml

```diff
@@ -6,17 +6,29 @@
                  t-on-click="() => this.trigger('click-product', props.product)"
                  t-att-data-product-id="props.product.id"
                  t-attf-aria-labelledby="article_product_{{props.product.id}}">
-            <div class="product-img">
-                <img t-att-src="imageUrl" t-att-alt="props.product.display_name" />
-            </div>
-            <div class="product-content">
-                <div class="product-name" t-attf-id="article_product_{{props.product.id}}">
-                    <t t-esc="props.product.display_name" />
+            <t t-if="props.product.image_128" >
+                <div class="product-img">
+                    <img t-att-src="imageUrl" t-att-alt="props.product.display_name" />
                 </div>
-                <span class="price-tag">
-                    <t t-esc="price" />
-                </span>
-            </div>
+                <div class="product-content">
+                    <div class="product-name" t-attf-id="article_product_{{props.product.id}}">
+                        <t t-esc="props.product.display_name" />
+                    </div>
+                    <span class="price-tag">
+                        <t t-esc="price" />
+                    </span>
+                </div>
+            </t>
+            <t t-else="">
+                <div class="product-content">
+                    <div class="product-name no-image" t-attf-id="article_product_{{props.product.id}}">
+                        <t t-esc="props.product.display_name" />
+                    </div>
+                    <span class="price-tag">
+                        <t t-esc="price" />
+                    </span>
+                </div>
+            </t>
```

Comment:
I think we should just do something like this:
`<t t-if="props.product.image_128">
                <div class="product-img">
                    <img t-att-src="imageUrl" t-att-alt="props.product.display_name" />
                </div>
            </t>`
and then instead of using t-else just for the class and use almost the 2 same blocks use this:
`<div t-attf-class="#{props.product.image_128 ? 'product-name' : 'product-name no-image' }"
                     t-attf-id="article_product_{{props.product.id}}">
                    <t t-esc="props.product.display_name" />
                </div>`